api/tests/unit/domain/usecases/find-user-for-oidc-reconciliation_test.js

```diff
@@ -88,7 +81,6 @@ describe('Unit | UseCase | find-user-for-oidc-reconciliation', function () {
   context('when authentication key is expired', function () {
     it('should throw an AuthenticationKeyExpired', async function () {
       // given
```

Comment:
Suggestion : Y'a un test qui s'appelle `it('should return authentication methods and full names'`, qui teste ça
` expect(authenticationSessionService.update).to.be.calledOnceWith('authenticationKey', sessionContentAndUserInfo);`
Je trouve que ça serait cool de l'expliciter que ça va mettre à jour la session content, du genre
it('should return authentication methods and full names and update user session content')

Reply:
idem, doublon